docker-compose.yml

```diff
@@ -11,20 +11,26 @@ services:
     image: redis
     ports:
       - '127.0.0.1:6379:6379'
+  celery:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file: common.env
```

Comment:
Does Celery require all of the env variables that Django app uses? If was wondering why to move them to separate file `common.env`.

Reply:
That's a good question, I assumed it would be better to have same environment for both django app and celery. I did move all variables to .env file to reuse them and simplify potential multi docker-compose configuration, ex. running all containers vs. without django app.
This way we can provide more than one env and more than one docker config, seems to be more flexible.

Reply:
@patrys What do you think?

Reply:
I think I'd prefer them in the compose file purely not to clutter the main directory any further but that's mainly a question of taste. I see both ways as viable and I think the variables are not likely to change (as this is not meant to build production environments).